Bottom: 80d69cf40c648ee0624daae7424d802e401eb813
Top:    e208442f48ccfb5e3870ec7aab8667e7ae6e95af
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-31 00:27:43 -0400

ADC register defines


---

diff --git a/arch/arm/mach-lpc31xx/include/mach/registers.h b/arch/arm/mach-lpc31xx/include/mach/registers.h
index 45af7fc..01f4114 100644
--- a/arch/arm/mach-lpc31xx/include/mach/registers.h
+++ b/arch/arm/mach-lpc31xx/include/mach/registers.h
@@ -276,6 +276,14 @@
  * ADC_REG register definitions
  **********************************************************************/
 #define ADC_CON_REG            __REG (ADC_PHYS + 0x20)
+#define ADC_R0_REG             __REG (ADC_PHYS + 0x00)
+#define ADC_R1_REG             __REG (ADC_PHYS + 0x04)
+#define ADC_R2_REG             __REG (ADC_PHYS + 0x08)
+#define ADC_R3_REG             __REG (ADC_PHYS + 0x0C)
+#define ADC_CSEL_REG           __REG (ADC_PHYS + 0x24)
+#define ADC_INT_ENABLE_REG     __REG (ADC_PHYS + 0x28)
+#define ADC_INT_STATUS_REG     __REG (ADC_PHYS + 0x2C)
+#define ADC_INT_CLEAR_REG      __REG (ADC_PHYS + 0x30)
 
 /***********************************************************************
  * SYS_REG register definitions
